src/playground/addons/story-utils/index.js

```diff
@@ -539,95 +539,113 @@ export const correctSvgPath = (data) => {
+      category: 'Content',
     },
-    required_text: {
-      name: 'required text',
-      type: 'string',
-      defaultValue: data.required_text,
-      description: 'Text to be shown when the form element is mandatory',
-      table: {
-        type: { summary: 'string' },
-        defaultValue: { summary: '*' },
-        category: 'Content',
-      },
+  };
+  argTypes.required_text = {
+    name: 'required text',
+    type: 'string',
+    defaultValue: data.required_text,
+    description: `Text to be shown when the form ${type} is mandatory`,
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '*' },
+      category: 'Content',
     },
-    invalid: {
-      name: 'invalid',
-      type: 'boolean',
-      defaultValue: data.invalid,
-      description: 'Marks the form element as invalid',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-      },
+  };
+  argTypes.invalid = {
+    name: 'invalid',
+    type: 'boolean',
+    defaultValue: data.invalid,
+    description: `Marks the form ${type} as invalid`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
     },
-    disabled: {
-      name: 'disabled',
-      type: 'boolean',
-      defaultValue: data.disabled,
-      description: 'Disabled form element',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-        disable: type !== 'element',
-      },
+  };
+  argTypes.disabled = {
+    name: 'disabled',
+    type: 'boolean',
+    defaultValue: data.disabled,
+    description: `Disabled form ${type}`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
+      disable: type !== 'element',
     },
-    required: {
-      name: 'required',
-      type: 'boolean',
-      defaultValue: data.required,
-      description: 'Sets the required attribute on the form element',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-      },
+  };
+  argTypes.required = {
+    name: 'required',
+    type: 'boolean',
+    defaultValue: data.required,
+    description: `Sets the required attribute on the form ${type}`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
     },
   };
+
+  if (data.width) {
+    argTypes.width = {
+      name: 'width',
+      type: { name: 'select' },
+      defaultValue: data.width,
+      description: `The width of the form ${type} {s: small, m: medium, l: large}`,
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: data.width },
+        category: 'Styles',
+      },
+      control: {
+        type: 'select',
+        options: ['s', 'm', 'l'],
```

Comment:
i see that you added a legend for this, still i feel that it would be better to define the labels in the select in an human readable way, so keeping the legend in place but having "small" "medium" and "large" as labels for the options.

Reply:
I was wondering the same, but I don't see how to define select control options with a specific label and it's value.
This is why I added them in the legend.